content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/cloud-account-associations/index.md

```diff
@@ -260,10 +261,144 @@ chainctl iam groups remove-aws
 
 After running either of these examples, `chainctl` will prompt you to select the Chainguard Enforce group whose association you'd like to remove. After selecting the group, `chainctl` will immediately remove the association.
 
-> Note: As of this writing, it is not possible to remove cloud account associations through the Chainguard Enforce Console.
+> Note: As of this writing, it is not possible to remove cloud account associations through the Chainguard Enforce Console. Also note that using `chainctl` to remove account associations will not remove any AWS IAM roles, GCP service accounts, or workload identity federation settings from your cloud provider.
 
 
 ## Learn More
 
 After setting up a cloud account association, you can use Chainguard Enforce's [Discovery feature](https://edu.chainguard.dev/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/chainguard-enforce-discovery-onboarding/) to discover various containerized workloads within your project. You can also set up an [Agentless Connection](https://edu.chainguard.dev/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/how-to-connect-kubernetes-clusters/#agentless-connections).
 
+## Additional IAM Resources
+
+In addition to the IAM roles or service accounts that Enforce creates for account associations, it also enables/requires the following resources:
+
+|Provider|Resource Type|Display Name|
+|--------|-------------|------------|
+|Google  |serviceusage.Service|storage-component.googleapis.com|
+|Google  |iam.Role     |Custom Role Chainguard Signer CA |
+|AWS     |AWS::IAM::OIDCProvider | ChainguardIDP |
+
+You can examine the resources that Enforce creates using the `gcloud` or `aws` command line tools depending on your cloud provider.
+
+### Google IAM Role
+
+Enforce creates an additional IAM role that allows listing certificates from a CA. You can examine the role with the following command:
+
+```
+gcloud iam roles list --project <your project name>
+```
+```
+---
+description: Chainguard signer role to list certificates from a CA
+etag: BwX4lw5KpDY=
+name: projects/<your project name>/roles/chainguardSignerCA
+title: Custom Role Chainguard Signer CA
+```
+
+### Google Workload Identity Pool
+
+Since Enforce uses OIDC for authentication, it relies on Workload Identity Federation to generate short-lived tokens. You can examine the identity pool, and the provider that Enforce creates with the following commands:
+
+```
+ gcloud iam workload-identity-pools list --location=global --project <your project name>

```

Comment:
This hangs off / requires horizontal scrolling, I would recommend line breaks `\`

Reply:
Good catch, will add everywhere